docs/tutorials/get-started/servo-mousemover.md

```diff
@@ -175,7 +175,9 @@ Create a [board component](/components/board/):
 
 1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
 1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-1. Select the `board` type, then select the `pi` model.
+1. Select the `board` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+   If you are using a Raspberry Pi 5, user the `pi5` model.
```

Comment:
```suggestion
   If you are using a Raspberry Pi 5, use the `pi5` model.
```
is this accurate? there is no servo pi5 model

Reply:
yes, that's correct.